turbopack/crates/turbopack-ecmascript-runtime/js/src/nodejs/runtime.ts

```diff
@@ -133,15 +133,13 @@ function loadChunkAsync(
       const chunkModules: CompressedModuleFactories = require(resolved)
       installCompressedModuleFactories(chunkModules, 0, moduleFactories)
       entry = loadedChunk
-    } catch (e) {
+    } catch (cause) {
       const errorMessage = `Failed to load chunk ${chunkPath} from module ${this.m.id}`
+      const error = new Error(errorMessage, { cause })
+      error.name = 'ChunkLoadError'
 
       // Cache the failure promise, future requests will also get this same rejection
-      entry = Promise.reject(
-        new Error(errorMessage, {
-          cause: e,
-        })
-      )
+      entry = Promise.reject(error)
```

Comment:
```suggestion
      entry = Promise.resolve().then(() => { throw error })
```

This is important as `Promise.reject` leads to a unhandled rejection even with `catch`

Reply:
that was a preexisting problem though then? I didn't change that part?